frontend/src/libs/components/components.ts

```diff
@@ -7,5 +7,6 @@ export { Loader } from './loader/loader.js';
 export { ProtectedRoute } from './protected-route/protected-route.js';
 export { RouterProvider } from './router-provider/router-provider.jsx';
 export { Sidebar } from './sidebar/sidebar.js';
+export { Controller } from 'react-hook-form';
```

Comment:
hey, if you update your branch and resolve the conflicts - you'll have form.type.ts file with react-hook-form exports, so you can add Controller there

Reply:
We aren't even allowed to use Controller, only `control`)

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/8f04ba0e-99ee-4a5d-94b3-9196f3651ca1)